classes/WP_CLI/Action/Create.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+use function \WP_CLI\Utils\get_flag_value;
+
+/**
+ * WP-CLI command: action-scheduler action create
+ */
+class ActionScheduler_WPCLI_Action_Create_Command extends ActionScheduler_WPCLI_Command {
+
+	const ASYNC_OPTS = array( 'async', 'now', 0 );
+
+	/**
+	 * Execute command.
+	 *
+	 * @uses as_schedule_single_action()
+	 * @uses as_enqueue_async_action()
+	 * @uses as_schedule_recurring_action()
+	 * @uses as_schedule_cron_action()
+	 * @uses $this->print_error()
+	 * @uses $this->print_success()
+	 * @return void
+	 */
+	public function execute() {
+		$hook           = $this->args[0];
+		$schedule_start = $this->args[1];
+		$callback_args  = get_flag_value( $this->assoc_args, 'args', array() );
+		$group          = get_flag_value( $this->assoc_args, 'group', '' );
+		$interval       = absint( get_flag_value( $this->assoc_args, 'interval', 0 ) );
+		$cron           = get_flag_value( $this->assoc_args, 'cron', '' );
+		$unique         = get_flag_value( $this->assoc_args, 'unique', false );
+		$priority       = absint( get_flag_value( $this->assoc_args, 'priority', 10 ) );
+
+		if ( ! empty( $callback_args ) ) {
+			$callback_args = json_decode( $callback_args, true );
+		}
+
+		$function_args = array(
+			'start'         => 'async',
+			'cron'          => $cron,
+			'interval'      => $interval,
+			'hook'          => $hook,
+			'callback_args' => $callback_args,
+			'group'         => $group,
+			'unique'        => $unique,
+			'priority'      => $priority,
+		);
+
+		// Generate schedule start if appropriate.
+		if ( ! in_array( $schedule_start, static::ASYNC_OPTS, true ) ) {
+			$schedule_start         = as_get_datetime_object( $schedule_start );
```

Comment:
`as_get_datetime_object` should be wrapped in a try--catch, mentioned that we were not able to parse the date (and possibly displaying an example or two)